Modelplace_Cloud_API/Object_detection/README.md

```diff
@@ -0,0 +1,189 @@
+# Object detection
+
+In this block, using the example of the [Faster R-CNN detector], we will describe in detail the steps of model integration into the Python application using [Modelplace Cloud API]. 
+
+To easily follow along with this tutorial, please download the [full code] and a [test image] and put these files in the same folder.   
+To run the script, run the command: 
+``` bash
+python object_detection.py 
+```
+
+
+## Step 1: Import the required libraries and set default parameters
```

Comment:
We don't need to have the steps here. We should explain how to run the code instead and use these texts as comments in the code